Exiled.API/Features/Player.cs

```diff
@@ -1933,15 +1990,37 @@ public void Kill(string deathReason, string cassieAnnouncement = "")
             ReferenceHub.playerStats.KillPlayer(new CustomReasonDamageHandler(deathReason, float.MaxValue, cassieAnnouncement));
         }
 
+        /// <summary>
+        /// Kills the player and vaporizes the body.
+        /// </summary>
+        /// <param name="attacker">The <see cref="Player"/> attacking player.</param>
+        /// <param name="cassieAnnouncement">The cassie announcement to make upon death.</param>
+        public void Vaporize(Player attacker = null, string cassieAnnouncement = "")
+        {
+            if ((Role.Side != Side.Scp) && !string.IsNullOrEmpty(cassieAnnouncement))
+                Cassie.Message(cassieAnnouncement);
+
+            Kill(new DisruptorDamageHandler(attacker?.Footprint ?? Footprint, -1));
```

Comment:
No need to use -1

Reply:
What should I set the damage amount to?

Reply:
Not adding damage argument would just make -1 no ?

Reply:
It's not optional